app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/PasscodeViewModel.kt

```diff
@@ -37,28 +38,28 @@ class PasscodeViewModel
 
     fun onForgotPasscode() {
         safeLaunch {
-
-            encryptionManager.removePassword()
-            encryptionManager.lock()
-            settingsHandler.usePasscode = false
-
             credentialsRepository.owners().forEach {
                 credentialsRepository.removeOwner(it)
+                notificationRepository.unregisterOwner()
```

Comment:
we can call unregisterOwner one time outside the loop. It is better to rename it to notificationRepository.unregisterOwners()

Reply:
yes